docs/details/clickhouse.md

```diff
@@ -0,0 +1,41 @@
+# ClickHouse
+
+
+You can use an external ClickHouse database instance outside the PMM Server container running on other hosts.
+
+## Environment variables
+
+PMM predefines certain flags that allow you to use ClickHouse parameters as environment variables:
+
+To use ClickHouse as an external database instance, use the following environment variables: 
+ 
+`PERCONA_TEST_PMM_CLICKHOUSE_ADDR` -> hostname:port
+:   Name of the host and port of the external ClickHouse database instance. 
+
+**Optional environment variables**
+
+`PERCONA_TEST_PMM_CLICKHOUSE_DATABASE` -> database name
+:   Database name of the external ClickHouse database instance.
+
+`​​PERCONA_TEST_PMM_CLICKHOUSE_POOL_SIZE` -> [pool size](https://github.com/ClickHouse/ClickHouse/blob/master/programs/server/config.xml#L1130)
```

Comment:
We have `database name` without square brackets and this and max_block_size with